tools/c7n_tencentcloud/c7n_tencentcloud/utils.py

```diff
@@ -13,3 +14,17 @@ class PageMethod(Enum):
     """
     Offset = 0
     PaginationToken = 1
+
+
+def isoformat_date_str(data: dict,
+                       field_keys: list,
+                       original_date_str_format: str,
+                       timezone_from=pytz.timezone("Asia/Shanghai"),
+                       timezone_to=pytz.utc):
+    """
+    standardize the date string, using isoformat including timezone info
+    example: '2022-09-28T15:28:28+00:00'
+    """
+    for key in field_keys:
+        dt = timezone_from.localize(datetime.strptime(data[key], original_date_str_format))
```

Comment:
date timezone from tencentcloud is +0800, but string format is not always the same, and most of them without tz info.
For safety, we just convert it to utc. Maybe it's not necessary, we can modify it.

Reply:
if we don't know the origin timezone, we can't convert without ambiguity/correctness concerns. hardcoding it to Shanghai tz isn't right, that could just be an artifact of which region/endpoint is being targeted. we should also try to capture on resource type metadata wrt to which services are returning local tz (either on regional basis) vs utc. it feels like this is a point where it would be good to engage with the service team to have a clearer understanding of the behavior.